### DISCUSSION TITLE
Inheritance in postbuild substitute

### DISCUSSION THREAD

**Discussion by siwyroot** [fluxcd/flux2#4707] [Q&A] [answered]
Hello,  dose postBuild variables are inherited to next Kustomization?  Kustomization A will have va1 and it will apply Kustomization B with var2, will var1 also be available at Kustomization B?

```
Kustomization A
    var1: foo
        Kustomization B
            var1: foo <- from Kustomization A
            var2: bar 
```

**Comment by stefanprodan** [comment, marked as answer]
No, Flux Kustomizations don't support inherited and never will, each object is reconciled independently. To share the same variables in multiple Kustomization, use `substituteFrom` and a ConfigMap/Secret. Docs here: https://fluxcd.io/flux/components/kustomize/kustomizations/#post-build-variable-substitution

**Comment by siwyroot** [comment]
Thanks Stefan!
